python/src/aiconfig/editor/client/src/components/ParametersRenderer.tsx

```diff
@@ -100,7 +101,7 @@ const ParameterInput = memo(function ParameterInput(props: {
         />
         <Textarea
           placeholder="Enter parameter value"
```

Comment:
Not related to this diff, but cc @saqadri why is paramater name `TextInput` while parameter value is `TextArea`?

Reply:
TextInput is intended for shorter text (like one line), text area is for larger text (multiple lines, expands)